client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -43,7 +43,11 @@ const DnsList = React.createClass( {
 	addDns: function( record ) {
 		addDnsAction( this.props.selectedDomainName, record, ( error ) => {
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'The DNS record could not be restored.' ) );
+			} else {
+				this.props.successNotice( this.translate( 'The DNS record has been restored.' ), {
+					duration: 5000
```

Comment:
Hi @klimeryk, go to sleep! :D 
So, honestly, `successNotice` shouldnt be used here either. I had a chat with @mtias about this and we kind of think that it would be best to render `<Notice>` component somewhere where it is most appropriate visually.
The reason i didnt use `props.errorNotice` here yet is that I dont understand what is in `error.message`.
If it so happens there will be some callbacks, components rendered inside a notice and other functionalities that new redux notices dont provide, we will have a problem.

SO:

1. Best to use Notice component for the stuff that makes sense locally
2. If above is too hard or does make sense globally, then use errorNotice action from redux.

I intended to implement 2, but I was lacking understanding the shape of `messages` object :)